client/src/main/java/io/cloudsoft/winrm4j/client/WinRmClient.java

```diff
@@ -737,6 +749,9 @@ public DeleteResponse call() {
                 assertFaultCode(soapFault, WSMAN_FAULT_CODE_SHELL_WAS_NOT_FOUND);
             }
         }
+        if (bus != null) {
```

Comment:
It is definitely good to have a fix for shutting down CXF bus.
However in CXF docs it is said that wsdl is taking care of creating a bus and thus sharing a common bus.
I suggest parametrizing shutdown on disconnect and not using it by default in Apache Brooklyn.
@neykov can you share performance results with this change?